src/hooks/useModal.ts

```diff
@@ -0,0 +1,17 @@
+import { modalPropsAtom } from '@/atoms/modalProps.atom'
+import { useSetRecoilState } from 'recoil'
+
+export const useModal = () => {
+  const setModalProps = useSetRecoilState(modalPropsAtom)
+  const onClose = () => {
+    setModalProps((prev) => {
+      return { ...prev, isOpen: false }
+    })
```

Comment:
```suggestion
    setModalProps((prev) => ({ ...prev, isOpen: false }))
```

불필요하게 return 문 쓰지 않아도 될 거 같습니다~

Reply:
- [refactor: 불필요하게 명시된 'return' 삭제](https://github.com/2023-chambit-project/KOOSMOOS/pull/64/commits/34b287d0cd6a8975933d1255ea3bc0c942886b0e)